modules/solid_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -735,7 +739,11 @@ DomainIntegralAction::act()
         params.set<std::vector<OutputName>>("outputs") = {"none"};
 
       if (_use_crack_front_points_provider && _used_by_xfem_to_grow_crack)
+      {
         params.set<ExecFlagEnum>("execute_on") = {EXEC_TIMESTEP_END, EXEC_NONLINEAR};
+        // The CrackFrontDefinition updates this vpp and MUST execute before it
+        params.set<int>("execution_order_group") = 1;
```

Comment:
Is the default value of that 0? Since you're already setting execution order -1 for CrackFrontDefinition, do you really need to set this one to 1, or is there some other order dependence?

Reply:
I removed it and added documentation to DomainIntegralAction about the execution order